src/main/webapp/ui/package-lock.json

```diff
@@ -3505,7 +3505,7 @@
       "version": "1.0.11",
       "resolved": "https://registry.npmjs.org/@mapbox/node-pre-gyp/-/node-pre-gyp-1.0.11.tgz",
       "integrity": "sha512-Yhlar6v9WQgUp/He7BdgzOz8lqMQ8sU+jkCq7Wx8Myc5YFJLbEe7lgui/V7G1qB1DJykHSGwreceSaD60Y0PUQ==",
-      "devOptional": true,
+      "optional": true,
```

Comment:
Just want to highlight all of these changes `devOptional` -> `optional` as it looks like the kind of thing that could sneak in unintentionally

Reply:
@rlamacraft can you confirm that change is due?

Reply:
I've re-run `npm install --force`, whichever changes persist are required